web/tailwind.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
       colors: {
         'red-25': '#FDF7F6',
         'greyscale-1': '#FBFBFF',
+        'greyscale-1.5': '#F4F4FB',
```

Comment:
Hm, is this used? (If not, possibly remove it -- I'm a bit skeptical of adding in-between shades but it's your call)

Reply:
I've been wanting it for a while, the difference between greyscale-1 and greyscale-2 is a bit too jarring. Yeah 1.5 is kind of annoying but I didn't want to shift everything over